aspnet/fundamentals/hosting.rst

```diff
@@ -0,0 +1,298 @@
+:version: 1.0.0
+
+Hosting
+=======
+
+By `Steve Smith`_
+
+To run an ASP.NET Core app, you need to configure and launch a host using ``WebHostBuilder``.
+
+.. contents:: Sections:
+  :local:
+  :depth: 1
+
+`View or download sample code <https://github.com/aspnet/Docs/tree/master/aspnet/fundamentals/hosting/sample>`__
+
+What is a Host?
+---------------
+
+ASP.NET Core apps require a *host* in which to execute. A host must implement the :dn:iface:`~Microsoft.AspNetCore.Hosting.IWebHost` interface, which exposes collections of features and services, and a ``Start`` method. The host is typically created using an instance of a :dn:class:`~Microsoft.AspNetCore.Hosting.WebHostBuilder`, which in builds and returns a  :dn:class:`~Microsoft.AspNetCore.Hosting.WebHost` instance. The ``WebHost`` has a private ``Server`` property that is used once the application is up and running to handle requests. Learn more about :doc:`servers <servers>`.
```

Comment:
webhost doesn't have a server property. At least I can't find it. Can you use a link there?